helm-chart/renku-gateway/templates/configmap.yaml

```diff
@@ -86,7 +86,7 @@ data:
           # This is to access undocumented APIs in GitLab
           entryPoints = ["http"]
           Middlewares = [
-            "direct"
+            "auth-gitlab","direct"
```

Comment:
Are those APIs we're using here still undocumented? Yet they do authentication with the users oauth token? It's a bit scary to rely on this 😬

Reply:
No, the authentication is for GrpahQL. Maybe we should have a separate route for that?